Makefile

```diff
@@ -1,16 +1,36 @@
-ROOT_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+ROOT_DIR := $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
```

Comment:
this one removes the `[...]/retis//retis/src/bindings` in the path (cosmetic). I put it here, but can be part of any PR/patch. Maybe I can move it in the outstanding Makefile fixes series?

The `out_console` is needed here because otherwise `gen-bindings V=1` will fail. 
Maybe there's a better way, but my Makefile-fu didn't suggest anything better :)